agent_build_refactored/managed_packages/managed_packages_builders.py

```diff
@@ -421,6 +423,8 @@ def _build_agent_package(
                 "--deb-no-default-config-files",
                 "--no-deb-auto-config-files",
                 "--config-files", "/etc/scalyr-agent-2/agent.json",
+                "--config-files", "/etc/scalyr-agent-2/agent.d",
```

Comment:
I guess the `agent.d` also has to be a config, but it isn't even in the current versions of the agent.